sympy/solvers/ode/single.py

```diff
@@ -425,17 +434,15 @@ def _get_general_solution(self, *, simplify: bool = True):
         return [gensol]
 
 
-
 class AlmostLinear(SinglePatternODESolver):
     r"""
     Solves an almost-linear differential equation.
 
     The general form of an almost linear differential equation is
 
-    .. math:: f(x) g(y) y + k(x) l(y) + m(x) = 0
-                \text{where} l'(y) = g(y)\text{.}
+    .. math:: a(x) g'(f(x)) f'(x) + b(x) * g(f(x)) + c(x) = 0
 
-    This can be solved by substituting `l(y) = u(y)`.  Making the given
+    This can be solved by substituting `f(x) = u`.  Making the given
```

Comment:
The substitution is really `u(x) = g(f(x))` isn't it? Also I think it's best to show what the substitution looks like in terms of a, b, and c. There should be no `*` in the math mode equation above.

Reply:
> The substitution is really `u(x) = g(f(x))` isn't it? 

Sorry for the error. I've fixed it now.